src/containerapp/azext_containerapp/_utils.py

```diff
@@ -1565,6 +1565,40 @@ def list_environment_locations(cmd):
     return res_locations
 
 
+# normalizes workload profile type
+def get_workload_profile_type(cmd, name, location):
+    return name.upper()
+
+
+def get_default_workload_profile(cmd, location):
```

Comment:
how about change to `get_default_workload_profile_name_from_env` ?

Reply:
changed get_default_workload_profile_from_env to get_default_workload_profile_name_from_env